bin/zf-development-mode

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env php
+<?php
+/**
+ * @link      http://github.com/zfcampus/zf-development-mode for the canonical source repository
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZF\DevelopmentMode;
+
+// Setup/verify autoloading
+if (file_exists($a = dirname(__FILE__).'/../../../autoload.php') === true) {
```

Comment:
a single space around concatenation operator (is it not specified in PSR-2? I can see we have it in other places)

why we need `=== true` here ??